cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -66,7 +66,7 @@ func TestAddFlags(t *testing.T) {
 		"--audit-log-truncate-enabled=true",
 		"--audit-log-truncate-max-batch-size=45",
 		"--audit-log-truncate-max-event-size=44",
-		"--audit-log-version=audit.k8s.io/v1alpha1",
```

Comment:
revert this, this is testing that explicitly setting the flag can override the default